sysml/src/validation/05-State-based Behavior/5-State-based Behavior-1a.sysml

```diff
@@ -83,7 +83,7 @@ package '5-State-based Behavior-1a' {
 					first off
 					accept 'Vehicle Start Signal' 
 					if vehicle1_c1.'brake pedal depressed'
-					do send 'Start Signal'() to vehicle1_c1.vehicleController
+					do send new 'Start Signal'() to vehicle1_c1.vehicleController
```

Comment:
In line with the change, in the comment block below
`The notation "'Start Signal'()" constructs` should be changed to
`The notation "new 'Start Signal'()" constructs`

In the comment there is also a typo: `parhentheses` -> `parentheses`.

Reply:
Fixed.